src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/GetPileupSummaries.java

```diff
@@ -208,7 +224,12 @@ public Object onTraversalSuccess() {
             throw new UserException.BadInput("No variants in population vcf had an allele frequency (AF) field.");
         }
         final String sampleName = ReadUtils.getSamplesFromHeader(getHeaderForReads()).stream().findFirst().get();
-        PileupSummary.writeToFile(sampleName, pileupSummaries, outputTable);
+
+        try {
```

Comment:
I think this belongs in closeTool rather than onTraversalSuccess (in case the traversal fails)

Reply:
good point